src/core/styles/legacy-buttons.css

```diff
@@ -3,7 +3,7 @@
     @apply text-white bg-cool-black text-btn2 font-sans font-bold inline-block rounded p-btn;
     @apply hover:text-white hover:bg-neutral-1100;
     @apply active:text-white active:bg-neutral-1200;
-    @apply focus:text-white focus:bg-cool-black focus:outline-offset-0 focus:outline-none focus-visible:outline-4 focus-visible:outline-gui-focus;
+    @apply focus:text-white focus:bg-cool-black focus-base;
```

Comment:
This is because `focus-base` is the _base_ class. Other modifications are allowed on top of it, but `focus-base` is the common class set across all use cases.